californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
I think there is a misunderstanding. In Eclipse Orbit you can find the base distributions (httpclient and httpcomponents) with the OSGi meta-data added by someone who needed these base bundles for an Eclipse plug-in. 
Nevertheless I understand that the risk for incorrect transitive dependencies because of the Maven dependency resolution would be a risk for other projects. I also noticed that element-connector has a similar issue because netty-all does not contain OSGi meta-data although all single netty JARs contain the meta-data. 
I also changed the Bundle-SymbolicName to follow the OSGi best practices.

Let me know what you think.